### PR TITLE
PBT on jenkins example

### PR DESCRIPTION
**Pull request by neontty**
updating PBT page to have two nested example CICD workflows:
- github actions
- Jenkins

has minor edits to details on github actions page

### CODE REVIEW

**Comment by neontty** [comment]
leaving as a draft for the moment until I can get approval/disapproval to move HelloCICD example repo into public org which would merit a few extra links

**Comment by neontty** [comment]
@slyons only on review because I made some minor edits to the github actions and pbt pages that you may want to skim. Otherwise no need to deep review on your part

File: docs/metadata/prophecy-build-tool/pbt-jenkins.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+You should have access to:
+
+- a Git repo with an existing Prophecy Project
+- a Jenkins Server where you can create new pipelines and credentials
+
+### Jenkins plugins
+
+The following plugins were used for this example:
+
+- [Github Pull Request Builder](https://plugins.jenkins.io/ghprb/)
```
Comment (atbida) [inline docs/metadata/prophecy-build-tool/pbt-jenkins.md:55]: "The Jenkins project announced unresolved security vulnerabilities affecting the current version of this plugin" fyi

File: docs/metadata/prophecy-build-tool/pbt-jenkins.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+
+:::caution
+
+The DATABRICKS_HOST and DATABRICKS_TOKEN env variables must match the configuration of the Fabric we are attempting to deploy.
+
+:::
+
+2. ('prepare system') - Ensure python3 is available on worker nodes. You can skip this if your Jenkins nodes already have `python3-venv` installed.
+3. ('install pbt') - Ensure PBT and its dependencies are installed
+4. ('deploy') - use PBT to deploy the Databricks jobs for our chosen Fabric
```
Comment (atbida) [inline docs/metadata/prophecy-build-tool/pbt-jenkins.md:301]: I added some step numbers here, but...steps 5-7 seem incorrect or out of order, right @neontty

File: docs/metadata/prophecy-build-tool/pbt-jenkins.md (hunk 1 above)
Comment (neontty) [reply docs/metadata/prophecy-build-tool/pbt-jenkins.md:55]: I can put in a warning, but ultimately it's an example and users don't have to use these triggers to create their pipelines

File: docs/metadata/prophecy-build-tool/pbt-jenkins.md (hunk 2 above)
Comment (neontty) [reply docs/metadata/prophecy-build-tool/pbt-jenkins.md:301]: ugh. the precommit hook kind of garbled this a few times. I'm trying to mix numbered list with sub bullets. I will fix

**Review by atbida — APPROVED** [review]
made some minor comments, and as long as the list here is reformatted/clarified, then looks good!

### Deploy Pipeline - Explanation of Pipeline

File: docs/metadata/prophecy-build-tool/pbt-jenkins.md (hunk 1 above)
Comment (neontty) [reply docs/metadata/prophecy-build-tool/pbt-jenkins.md:55]: put in a warning

**Review by atbida — APPROVED** [review]

**Pull request merged**